env0/resource_environment.go

```diff
@@ -1443,7 +1443,7 @@ func resourceEnvironmentImport(ctx context.Context, d *schema.ResourceData, meta
 			return nil, fmt.Errorf("failed to get template with id %s: %w", templateId, err)
 		}
 
-		if err := templateRead("without_template_settings", template, d, true); err != nil {
+		if err := templateRead("without_template_settings", template, d); err != nil {
```

Comment:
There was special handling related to import and TerragruntTfBinary - it's no longer needed since we will implicitly default to "opentofu".

If there is a value returned from the http response (when askign for the template) it will be assigned to the schema.

In other words, the code that was handling a special use case related to terraform import and TerragruntTfBinary was removed. So it's no longer required to pass a true/false flag.